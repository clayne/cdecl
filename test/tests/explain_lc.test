cdecl @ @ @ explain long char x @ 65
