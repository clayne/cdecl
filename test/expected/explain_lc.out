explain long char x
        ^
9: error: "long char" is illegal for built-in type
